backend/server/controllers.js

```diff
@@ -101,7 +101,7 @@ module.exports = {
       .then((result) => res.status(200).json(result))
       .catch((err) => {
         console.log(err);
```

Comment:
Is this production grade? If so, remove console.logs on controllers.js file

Reply:
I would still like to keep error logging at this phase since we are still developing and cause it does help when things don't work. Will remove when everything is done.